src/devlink/NavbarComponent.js

```diff
@@ -88,22 +88,37 @@ export function NavbarComponent({ as: _Component = _Builtin.NavbarWrapper }) {
                 href: "#",
               }}
             >
-              {"Our impact"}
+              {"Greg's Stripe"}
             </_Builtin.NavbarLink>
           </_Builtin.Block>
           <_Builtin.Block
             className={_utils.cx(_styles, "navbar_menu-buttons")}
             tag="div"
+            data-stripe-donate="true"
           >
             <Button
               buttonText="Donate"
               buttonLink={{
-                href: "https://donate.stripe.com/test_eVq14pfCB9iW5MKcwx0VO03",
+                href: "https://donate.stripe.com/9B68wPaG61fm4Bwb4Z5J602",
```

Comment:
@brodel10 @cutiepatootiebear 
When setting the url in webflow it would override with the "https://donate.stripe.com/test_eVq14pfCB9iW5MKcwx0VO03".
 Now that you synced it, webflow picked up the correct href for the stripe integration.

 I am curious if it was set to an empty string would webflow have the ability to override